optimade/server/routers/info.py

```diff
@@ -44,7 +44,7 @@ def get_info(request: Request):
             id=BaseInfoResource.schema()["properties"]["id"]["const"],
             type=BaseInfoResource.schema()["properties"]["type"]["const"],
             attributes=BaseInfoAttributes(
-                api_version=f"v{__api_version__}",
+                api_version=f"{__api_version__}",
```

Comment:
```suggestion
                api_version=__api_version__,
```

Reply:
Think I caught this one already? Must have been between force pushes ;)